shared/bundle_analysis/report.py

```diff
@@ -76,40 +89,79 @@ class BundleReport:
     Report wrapper around a single bundle (many of which can exist in a single analysis report).
     """
 
-    def __init__(self, db_path: str, bundle: models.Bundle):
+    def __init__(self, db_path: str, bundle: Bundle):
         self.db_path = db_path
         self.bundle = bundle
 
     @property
     def name(self):
         return self.bundle.name
 
```

Comment:
This part is the only real change, everything else is import pathing change.
Basically adding to some sqlalchemy query conditions for the 3 filter options.